### PR TITLE
ELPP-2042 Recommendations SQS watch command

### PR DESCRIPTION
**Pull request by stephenwf**

### CODE REVIEW

File: src/Recommendations/Command/MysqlRepoQueueCommand.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        callable $limit
+    ) {
+        $this->rules = $rules;
+
+        parent::__construct($logger, $queue, $transformer, $monitoring, $limit);
+    }
+
+    protected function process(InputInterface $input, QueueItem $model)
+    {
+        $this->rules->import(new RuleModel($model->getId(), $model->getType()));
```
Comment (stephenwf) [inline src/Recommendations/Command/MysqlRepoQueueCommand.php]: @giorgiosironi @elrossco22 

After using the `QueueCommand` from bus sdk, these types of fairly complex long running processes become trivial!

File: src/App/Console.php [hunk 2]
```diff
@@ -33,6 +35,23 @@ public function __construct(Application $console, Kernel $app)
...
+    }
+
+    public function queueCreateCommand()
+    {
+        // aws sqs create-queue --region=us-east-1 --queue-name=eLife-recommendations --endpoint=http://elife_recommendations_sqs:4100
+        /** @var SqsClient $queue */
+        $queue = $this->app->get('aws.sqs');
+
+        $queue->createQueue([
+            'Region' => 'us-east-1',
```
Comment (giorgiosironi) [inline src/App/Console.php]: these should come from configuration

File: src/Recommendations/Command/MysqlRepoQueueCommand.php (hunk 1 above)
Comment (giorgiosironi) [inline src/Recommendations/Command/MysqlRepoQueueCommand.php]: I like the smell of abstraction in the morning

**Comment by stephenwf** [comment]
Rebased

**Pull request merged**